sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/CallingApplicationIdentifier.java

```diff
@@ -30,7 +30,27 @@ public CallingApplication(String id) {
      *
      * @return the string identifier representing the object identity
```

Comment:
Let's be more specific in our wording.

```suggestion
     * @return the string identifier representing the Unknown identifier
```